core/src/Borrower.sol

```diff
@@ -103,46 +102,50 @@ contract Borrower is IUniswapV3MintCallback {
             // Fetch liabilities from lenders
             (liabilities0, liabilities1) = _getLiabilities();
+            // Calculate liquidation incentive
+            (incentive1, priceX96) = BalanceSheet.computeLiquidationIncentive(
+                assets.fixed0 + assets.fluid0C, // total assets0 at `prices.c` (the TWAP)
+                assets.fixed1 + assets.fluid1C, // total assets1 at `prices.c` (the TWAP)
+                liabilities0,
+                liabilities1,
+                prices.c
+            );
             // Ensure only unhealthy accounts can be liquidated
-            require(!BalanceSheet.isHealthy(prices, assets, liabilities0, liabilities1), "Aloe: already healthy");
+            require(!BalanceSheet.isHealthy(prices, assets, liabilities0, liabilities1, incentive1), "Aloe: healthy");
         }
 
         // NOTE: The health check values assets at the TWAP and is difficult to manipulate. However,
         // the instantaneous price does impact what tokens we receive when burning Uniswap positions.
         // As such, additional calls to `TOKEN0.balanceOf` and `TOKEN1.balanceOf` are required for
         // precise inventory.
 
-        // Figure out what portion of `liabilities0` can be repaid using existing `assets0`
-        uint256 assets0 = TOKEN0.balanceOf(address(this));
-        uint256 repayable0 = Math.min(assets0, liabilities0);
+        // Figure out what portion of `liabilities0` can be repaid using existing assets
+        uint256 repayable0 = Math.min(liabilities0, TOKEN0.balanceOf(address(this)));
         unchecked {
             liabilities0 -= repayable0;
         }
 
-        // Figure out what portion of `liabilities1` can be repaid using existing `assets`.
-        uint256 assets1 = TOKEN1.balanceOf(address(this));
-        uint256 repayable1 = Math.min(assets1, liabilities1);
+        // Figure out what portion of `liabilities1` can be repaid using existing assets
+        uint256 repayable1 = Math.min(liabilities1, TOKEN1.balanceOf(address(this)));
         unchecked {
             liabilities1 -= repayable1;
         }
 
-        // TODO: this is already computed inside of computeLiquidationIncentive, so use that instead of
-        // re-computing it (or at least compare gas between the 2 options)
-        uint256 priceX96 = Math.mulDiv(prices.c, prices.c, Q96);
-
         if (liabilities0 + liabilities1 == 0 || (liabilities0 > 0 && liabilities1 > 0)) {
             // If both are zero or neither is zero, there's nothing more to do
             // TODO: compensate liquidators for txn costs using ANTE
         } else if (liabilities0 > 0) {
             uint256 converted0 = liabilities0 / strain;
 
-            uint256 maxLoss1 = Math.mulDiv(converted0, priceX96, Q96);
-            maxLoss1 += maxLoss1 / LIQUIDATION_INCENTIVE;
+            uint256 maxLoss1 = Math.mulDiv(converted0, priceX96, Q96) + incentive1 / strain;
```

Comment:
I'm not 100% on order of operations in solidity, but here are you trying to do $$\frac{Math.mulDiv(converted0, priceX96, Q96) + incentive1}{strain}$$? Or are you trying to $$Math.mulDiv(converted0, priceX96, Q96) + \frac{incentive1}{strain}$$? I feel like it would be the former because the `strain` determines how much the callee needs to swap in order to cover the liabilities and the liquidation incentive is a potential liability.

Reply:
So, that's all true -- only thing is that `converted0` already includes the division by `strain`, so here I'm doing the 2nd set of LaTeX.

Reply:
Actually let me try doing it all at once. Def would improve readability and now that I think about it, it may save us 3 gas as well (one opcode improvement lol)

Reply:
Curious if the variable naming change [here](https://github.com/aloelabs/aloe-ii/pull/40/files) changes your mind.

Reply:
@aneesh1 I suggest we table this for now, as we'll be more than able to address it once this slew of PRs is merged. I don't really want to edit this one, since everything else builds on it.